### DISCUSSION TITLE
[Docs] Comparison with Entity Framework

### DISCUSSION THREAD

**Discussion by Meligy** [linq2db/linq2db#4288] [General]
### Describe your issue

Can you please add some comparisons with Entity Framework being the defacto LINQ ORM in dotnet nowadays? Thanks a lot.

### Steps to reproduce

Go to the repo homepage, look for a comparison to Entity Framework.

Then go to documentation and look there too. Check the FAQ for example.

Nothing seems to be found.

### Environment details

`Linq To DB` version: N/A

Database (with version): N/A

ADO.NET Provider (with version): N/A

Operating system: N/A

.NET Version: 6+

**Comment by Meligy** [comment]
And just to be clear, I've found:

> However, it's not as heavy as LINQ to SQL or Entity Framework. There is no change-tracking, so you have to manage that yourself, but on the positive side you get more control and faster access to your data.

I'm hoping there's more to this, as EF can be used without tracking too.

**Comment by sdanyliv** [comment]
`linq2db` has more powerful LINQ queries support. What you cannot query via EF is easily doable via `linq2db`.
`Merge`, `Window Functions`, `FULL JOIN`, Temporary Tables, BulkInsert. and many many other things.
